x-pack/metricbeat/module/azure/_meta/config.yml

```diff
@@ -111,3 +111,11 @@
 #  api_key: ''
 #  metrics:
 #   - id: ["requests/count", "requests/duration"]
+
+#- module: azure
+#  metricsets:
+#    - app_state
+#  enabled: true
```

Comment:
Is `enabled: true` required here?

Reply:
not necessary , just making sure users are aware of this flag